server/views/assessments/pages/review-application/review.njk

```diff
@@ -0,0 +1,171 @@
+    <div class="govuk-width-container">
+      <h1 class="govuk-heading-xl">
+        {{ page.title }}
+      </h1>
+
+      {{ mojSubNavigation({
+        label: 'Sub navigation',
+        attributes: {
+          'role': 'tablist'
+        },
+        items: AssessmentUtils.getReviewNavigationItems()
+      }) }}
+
+      <div class="main-box">
+        <h3 class="box-title govuk-heading-m">
+          Person Details
+        </h3>
+        <div class="box-content" data-cy-review-section="person-details" >
+          {{ govukSummaryList({
+          rows: ([
+            {
+              key: {
+                text: "Names"
+              },
+              value: {
+                text: page.document.application.person.name
+              }
+            },
+            {
+              key: {
+                text: "CRN"
+              },
+              value: {
+                text: page.document.application.person.crn
+              }
+            },
+            {
+              key: {
+                text: "Date of Birth"
+              },
+              value: {
+                text: formatDate(page.document.application.person.dateOfBirth, { format: 'short' })
+              }
+            },
+            {
+              key: {
+                text: "NOMS Number"
+              },
+              value: {
+                text: page.document.application.person.nomsNumber
+              }
+            },
+            {
+              key: {
+                text: "Nationality"
+              },
+              value: {
+                text: page.document.application.person.nationality
+              }
+            },
+            {
+              key: {
+                text: "Religion or Belief"
+              },
+              value: {
+                text: page.document.application.person.religionOrBelief
+              }
+            },
+            {
+              key: {
+                text: "Sex"
+              },
+              value: {
+                text: page.document.application.person.sex
+              }
+            },
+            {
+              key: {
+                text: "Gender Identity"
+              },
+              value: {
+                text: page.document.application.person.genderIdentity
+              }
+            },
+            {
+              key: {
+                text: "Status"
+              },
+              value: {
+                html: statusTag(page.document.application.person.status)
+              }
+            },
+            {
+              key: {
+                text: "Prison"
+              },
+              value: {
+                text: page.document.application.person.prisonName
+              }
+            }
+          ])
+        }) }}
+        </div>
+      </div>
+
+      {% for section in AssessmentUtils.assessmentSections(page.document.application) %}
+        <h2 class="govuk-heading-l" id={{section.title | kebabCase }}>{{ section.title }}</h2>
```

Comment:
I don't have a good solution, and it's definitely not a must fix, but it makes me a little uncomfortable that multiple places need to just know that "kebabify the title" is how you get a section's anchor ID

Reply:
A good point. 
The best solution I can think of is wrapping the `kebabCase` function with a more explicitly named function so it's a bit more obvious as to what's going on.